lib/chalk/query.ex

```diff
@@ -74,18 +74,40 @@ defmodule Chalk.Query do
 
   @doc """
   Execute an online query
-  Parameters
-  ```
-  %{input: }
-  ```
   """
+  @spec online(
+          %{
+            required(:inputs) => %{
+              String.t() => String.t()
+            },
+            required(:outputs) => [String.t()],
+            optional(:staleness) => %{
+              String.t() => String.t()
+            },
+            optional(:context) => %{
+              optional(:environment) => String.t(),
+              optional(:tags) => [String.t()]
+            },
+            optional(:deployment_id) => String.t(),
+            optional(:query_name) => String.t()
+          },
+          map()
+        ) :: {:ok, OnlineQueryResponse.t()} | {:error, any()}
```

Comment:
@AndyMoreland @rileysiebel Any other keys we want to add here? Followed the docs [here](https://docs.chalk.ai/docs/query-basics#request).

Reply:
you could add `correlation_id` if you want to inject a consistent id into the logs we emit to your datadog, but not a Mandatory Field